packages/database/prisma/migrations/20241129202947_add_last_visited_workspace_id/migration.sql

```diff
@@ -0,0 +1,5 @@
+-- AlterTable
+ALTER TABLE "User" ADD COLUMN     "lastVisitedWorkspaceId" UUID;
+
+-- AddForeignKey
+ALTER TABLE "User" ADD CONSTRAINT "User_lastVisitedWorkspaceId_fkey" FOREIGN KEY ("lastVisitedWorkspaceId") REFERENCES "Workspace"("id") ON DELETE SET NULL ON UPDATE CASCADE;
```

Comment:
Can we make this migration set the first value of `lastVisitedWorkspaceId` as the _latest_ workspace this user created? If that doesn't exist, we could just set it to the _earliest_ workspace.

Reply:
The source code will default to one he is owner of